### ISSUE TITLE
TimePicker does not show AM/PM or inner circle hours

### ISSUE THREAD

**Issue by kornelijepetak** [toggl/mobileapp#2336]
When Start/Stop/Duration popup opens, the 24 hour mode is being set up from user preferences.
However, no matter what I tried, the outcome is always one of the following:

1. AM/PM mode: The AM/PM buttons are not visible (first screenshot) and can't be clicked.

2. 24 Hours mode: The inner circle of hours is not visible. The hours can be selected (second screenshot), but the labels are not visible. (If `SetIs24HourView` is called within `OnFinishInflate`, the inner circle shows, but this results in case 1. if it was the actual mode).

Basically it seems that change is not good.

What I've tried (to no avail):
- setting up the timer so that this mode is changed every second, the same problem happens
- set up to other mode and then set to the desired mode right after
- RequestLayout, Invalidation - nope.

![image](https://user-images.githubusercontent.com/7033216/40721298-63cd61b0-6419-11e8-8890-64402d3a9ca9.png)

![image](https://user-images.githubusercontent.com/7033216/40721294-5f30e992-6419-11e8-96ab-3bd343075910.png)
